src/modules/order/index.ts

```diff
@@ -53,29 +53,28 @@ export async function getOrderDetails (orderId: string): Promise<Order | Respons
 
       order.parts = orderShipGroup as OrderPart[]
 
-      response = order
+      response = Promise.resolve(order)
```

Comment:
We could return the success response as well

Reply:
Yes Sir, but suppose we want to peform some extra operation on success response before returning, that's why stored it in a variable.

Reply:
I think then we should perform operations on original object and then only resolve

Reply:
I think then we should first perform all the operations and then only resolve

Reply:
Ok Sir